meseq.py

```diff
@@ -330,14 +338,23 @@ def draw(self):
         y = 0
         for row in self.matrix.rows:
             y += STEP
-            for i in range(len(row)):
```

Comment:
I had to draw life lines first, otherwise, life lines were drawn on top of the text when arrows were coming from the left.